modules/core/test/test_rect.cpp

```diff
@@ -0,0 +1,20 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+#include "test_precomp.hpp"
+
+namespace opencv_test { namespace {
+
+    TEST(Rect, rectUnionOperator) {
```

Comment:
Intersection?

Reply:
I believe this test should be nearby similar ones.

Reply:
it should be intersection, I will change it, both here and in the pr title and issue title, sorry for the language mistaken